src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -12,6 +12,9 @@
 from ceph.deployment.utils import is_ipv6, unwrap_ipv6
 from orchestrator import OrchestratorError, DaemonDescription
 from cephadm import utils
+import secrets
+from mgr_util import create_self_signed_cert, ServerConfigException, verify_tls
+import collections
```

Comment:
`secrets` and `collections` belong to the list of import statements that import the stdlibrary above.

Reply:
OK

Reply:
https://www.python.org/dev/peps/pep-0008/#imports